frontends/web/package.json

```diff
@@ -52,7 +52,7 @@
   },
   "scripts": {
     "dev": "npm run start",
-    "start": "vite --port 8080 --cors",
+    "start": "vite --host --port 8080",
```

Comment:
Yes I did test that but thought as we did not have it accessible over IP before this PR it does not matter that it  does not work. The intention is to make it accessible when it runs in the container (`make dockerdev`).

But yeah I guess if we add `--host` we should also figure out how to make it accessible over ip then. I'll look into this.

Reply:
I don't really know enough about containers and can't really help here :/

Reply:
It's  a _CORS_ issue...we can change [this](https://github.com/digitalbitbox/bitbox-wallet-app/blob/192e22dd2e9e2bbbb9565ab40b77704c8aef295d/backend/handlers/handlers.go#L1137) line to allow any origin (only in dev mode):
` w.Header().Set("Access-Control-Allow-Origin", "*")`

Now you can run only frontend/backend in the container or both of them and it will work. However the vite server will tell you it's running on `10....` now but you will not be able to access that from there as it is not your network but the containers network (that we can reach on either local host or `192..` because we exported the ports)
![image](https://github.com/digitalbitbox/bitbox-wallet-app/assets/90551797/f5d4365c-17e8-4516-a2d6-136ef76191ab)

Reply:
I think there is no fix for that because we already can reach the container from local host and even from local network, he address vite gives us ist just wrong because it is in the container so for it the network is a different one than ours.